python/ccxt/pro/test/test_order_book.py

```diff
@@ -1,396 +0,0 @@
-import os
-import sys
-
-root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
-sys.path.append(root)
-
-# ----------------------------------------------------------------------------
-
-# PLEASE DO NOT EDIT THIS FILE, IT IS GENERATED AND WILL BE OVERWRITTEN:
-# https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code
-
-# ----------------------------------------------------------------------------
-
-from ccxt.async_support.base.ws.order_book import OrderBook, IndexedOrderBook, CountedOrderBook  # noqa: F402
-
-
-def equals(a, b):
```

Comment:
on the newest version we get this version of equals, which does not seem right 

![image](https://github.com/ccxt/ccxt/assets/43336371/4c262060-7eda-4875-8724-4ad61495a612)

Reply:
same thing in php

Reply:
can you plz elaborate on those functions? they have been there before and i dont actually know what they do there (havent touched and havent checked till date).
what should I do to them?

Reply:
ah wait, len(a) len(b) ?? ogrhhh

Reply:
fixed in both php & py

Reply:
if any other notes, lmk